include/nds/arm9/math.h

```diff
@@ -254,14 +254,14 @@ u32 sqrt64(long long a)
 
 static inline
 /**
-*   \brief 1.19.12 fixed point cross product function result = AxB
+*   \brief 20.12 fixed point cross product function result = AxB
 *   \param a pointer to fixed 3x3 matrix
 *   \param b pointer to fixed 3x3 matrix
 *   \param result pointer to fixed 3x3 matrix
 * Cross product
```

Comment:
Added newline to line 261 I can also add them to the formulas in the brief statements, but I'll have to more those to the description, because \brief macro does not honour newline.